lib/internal/modules/esm/loader.js

```diff
@@ -41,19 +40,21 @@ const {
   validateObject,
   validateString,
 } = require('internal/validators');
-const ModuleMap = require('internal/modules/esm/module_map');
-const ModuleJob = require('internal/modules/esm/module_job');
+function newModuleMap() {
+  const ModuleMap = require('internal/modules/esm/module_map');
+  return new ModuleMap();
+}
 
 const {
   defaultResolve,
   DEFAULT_CONDITIONS,
 } = require('internal/modules/esm/resolve');
-const {
-  initializeImportMeta
-} = require('internal/modules/esm/initialize_import_meta');
-const { defaultLoad } = require('internal/modules/esm/load');
-const { translators } = require(
-  'internal/modules/esm/translators');
+
+function getTranslators() {
+  const { translators } = require(
+    'internal/modules/esm/translators');
+  return translators;
+}
```

Comment:
nit: I think the line-break is not necessary here

```suggestion
function getTranslators() {
  const { translators } = require('internal/modules/esm/translators');
  return translators;
}
```

But if it is

```suggestion
function getTranslators() {
  const { translators } = require(
    'internal/modules/esm/translators'
  );
  return translators;
}
```